index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link rel="stylesheet" href="app.css">
-    <title>Freddy KABERUKA</title>
+    <title>Freddy KABERUKA Portofolio</title>
```

Comment:
Kindly change the contrast to be at least `4.5` for these failed elements(`see project` on `hover` and ` Get in touch ` on `hover`) because they failed in the contrast check
![image](https://user-images.githubusercontent.com/24703171/161763653-13a8ac98-c268-4098-b13f-b97e43a5d1fb.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)